spring-webmvc/src/main/java/org/springframework/web/servlet/mvc/condition/ConsumesRequestCondition.java

```diff
@@ -16,20 +16,12 @@
 
 package org.springframework.web.servlet.mvc.condition;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
@sbrannen also I haven't seen this requirement in the code style ( https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style ) - worth adding and probably configuring codacy appropriately